src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UploadContestedCaseDocumentsService.java

```diff
@@ -0,0 +1,113 @@
+        applicantCorrespondenceCollection.addAll(correspondenceFiltered);
+        log.info("Adding item: {}, to Applicant Correspondence Collection", correspondenceFiltered);
+
+        List<ContestedUploadedDocumentData> frFormsFiltered = uploadedDocuments.stream()
+            .filter(d -> d.getUploadedCaseDocument().getCaseDocumentParty() != null
+                && d.getUploadedCaseDocument().getCaseDocumentParty().equals(APPLICANT))
+            .filter(d -> d.getUploadedCaseDocument().getCaseDocumentType().equals("Form B")
+                    || d.getUploadedCaseDocument().getCaseDocumentType().equals("Applicant - Form E")
+                    || d.getUploadedCaseDocument().getCaseDocumentType().equals("Form F")
+                    || d.getUploadedCaseDocument().getCaseDocumentType().equals("Form G")
+                    || d.getUploadedCaseDocument().getCaseDocumentType().equals("Form H"))
+            .collect(Collectors.toList());
+        List<ContestedUploadedDocumentData> applicantFormCollection = getDocumentCollection(caseData, APPLICANT_FR_FORM_COLLECTION);
+        applicantFormCollection.addAll(frFormsFiltered);
+        log.info("Adding item: {}, to Applicant FR Forms Collection", frFormsFiltered);
+
+        List<ContestedUploadedDocumentData> evidenceInSupportFiltered = uploadedDocuments.stream()
+            .filter(d -> d.getUploadedCaseDocument().getCaseDocumentParty() != null
+                && d.getUploadedCaseDocument().getCaseDocumentParty().equals(APPLICANT))
+            .filter(d -> d.getUploadedCaseDocument().getCaseDocumentType().equals("Statement of Issues")
+                    || d.getUploadedCaseDocument().getCaseDocumentType().equals("Chronology")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Case Summary")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Questionnaire")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Reply to Questionnaire")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Valuation Report")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Pension Plan")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Position Statement")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Skeleton Argument")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Expert Evidence")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Witness Statement/Affidavit")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Care Plan")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("Offers")
+                || d.getUploadedCaseDocument().getCaseDocumentType().equals("other"))
```

Comment:
would be more readable having a constant list at the top of the class
private final static List<String> VALID_TYPES= Arrays.asList("Statement of Issues", "Chronology" ....);
then in the filter
VALID_TYPES.contains(d.getUploadedCaseDocument().getCaseDocumentType())